src/main/java/com/openlattice/datastore/edm/controllers/EdmController.java

```diff
@@ -112,11 +115,18 @@
     @Inject
     private DataGraphManager dgm;
 
+    @Inject
+    private Environment env;
+
     @RequestMapping(
             path = CLEAR_PATH,
             method = RequestMethod.DELETE )
     @ResponseStatus( HttpStatus.OK )
     public void clearAllData() {
+        if ( !env.acceptsProfiles( TEST_PROFILE ) ) {
```

Comment:
I still think we should move this to a different controller and api, but good enough for now.